web/.htaccess

```diff
@@ -5,6 +5,10 @@
 # to each configured DirectoryIndex file (e.g. index.php, index.html, index.pl).
 DirectoryIndex app.php
 
+# Disable the MultiViews option preventing from using route paths beginning with
+# "/app", "/app_dev" or "/config".
```

Comment:
`Disabling MultiViews prevents unwanted negotiation, e.g. "/app" should not resolve to the front controller "/app.php" but be rewritten to "/app.php/app"